### ISSUE TITLE
Script w EmergencyMode does not run

### ISSUE THREAD

**Issue by realrelook** [quadpunk/ShellyTibberScript#4]
I tried the version from 14th.May. with EmergencyMode. The Script can run when you press start, but after receiving error msg it stops.
***error msgbelow***
Uncaught ReferenceError: "getEnergyPriceQuery" is not defined
22:31:13
at body: getEnergyPriceQuery()
22:31:13
^
22:31:13
in function "monitorPricesAndTriggerEmergency" called from monitorPricesAndTriggerEmergency();
22:31:13
^
22:31:13
in function called from system
22:31:13
***error msgEnd***

Could this be fixed.? I tried by myself a lot of versions with an additional GET, but all did not work. Maybe you have more luck than me.

**Comment by quadpunk** [comment]
Yes sorry i also didnt get the newer version to work now. I've reuploaded the older previously working version with a somewhat simpler logic.
